test/bigtest/tests/edit-order-test.js

```diff
@@ -31,20 +31,6 @@ describe('OrderEditPage', function () {
     expect(orderEditPage.notes().length).to.be.equal(1);
   });
 
-  it('displays owner', function () {
-    expect(orderEditPage.owner.value).to.equal(OWNER_TEST_VALUE);
```

Comment:
Also, you can remove `owner` from interactors `order-details-page.js` and `order-edit-page.js`, and from translations